Patch is project specific, and not a candidate for drupal contribution.
This patch removes an alter that prevents the moderation sidebar from appearing
when using our configuration of headless_lightning profile.
diff --git a/modules/json_content/json_content.module b/modules/json_content/json_content.module
index 9bc5497..83e33ce 100644
--- a/modules/json_content/json_content.module
+++ b/modules/json_content/json_content.module
@@ -1,5 +1,10 @@
 <?php

+/**
+ * @file
+ * Short comment.
+ */
+
 use Drupal\Core\Entity\EntityInterface;

 /**
@@ -11,7 +16,6 @@ function json_content_toolbar_alter(array &$items) {
   unset($items['home']);
 }

-
 /**
  * Implements hook_entity_operation_alter().
  */
@@ -24,31 +28,3 @@ function json_content_entity_operation_alter(array &$operations, EntityInterface
     ]);
   }
 }
-
-/**
- * Implements  hook_menu_local_tasks_alter().
- */
-function json_content_menu_local_tasks_alter(array &$data, $route_name) {
-  if (isset($data['tabs'][0])) {
-    foreach ($data['tabs'][0] as $name => $tab) {
-      if (preg_match('/^entity\.(\S*)\.canonical$/', $name, $matches)) {
-        // If there is a link to the canonical view of an entity, override it so
-        // that it points to the JSON rendered version of the entity. We tried
-        // doing this with hook_local_tasks_alter, but ended up with links to
-        // other bundles of the same entity types.
-        $entity_type = $matches[1];
-        $entity = \Drupal::routeMatch()->getParameter($entity_type);
-        $url = lightning_api_entity_json($entity);
-        $data['tabs'][0]['entity.' . $entity_type . '.canonical']['#link']['url'] = $url;
-        $data['tabs'][0]['entity.' . $entity_type . '.canonical']['#link']['title'] = t('View JSON');
-      }
-      if (preg_match('/^content_moderation\.workflows\:(\S*)\.latest_version_tab$/', $name, $matches)) {
-        // JSON API isn't capable of showing specific revisions, so we need to
-        // remove the Latest Revision tab if it exists. Once this issue is
-        // resolved we should be able to provide a link to the latest revision:
-        // https://www.drupal.org/project/jsonapi/issues/2795279
-        $data['tabs'][0]['content_moderation.workflows:' . $matches[1] . '.latest_version_tab']['#access'] = FALSE;
-      }
-    }
-  }
-}
\ No newline at end of file
